bellbot_action_server/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_action_server</name>
-  <version>0.0.10</version>
+  <version>0.0.11</version>
```

Comment:
Never change version numbers manually! This will not be releasable now. The version number is changed by Jenkins automatically and ALL packages in this repository have to have the same version number.